java/build.gradle

```diff
@@ -19,6 +19,8 @@ dependencies {
     implementation 'org.springdoc:springdoc-openapi-ui:1.2.32'
     implementation 'org.springdoc:springdoc-openapi-data-rest:1.2.32'
     implementation 'com.fasterxml.jackson.core:jackson-databind'
+    implementation 'org.projectlombok:lombok:1.18.12'
```

Comment:
Whilst you're at it we can probably just remove the gradle files for now to avoid confusion (and extra maintenance work)